services/nlp/src/main/java/ca/bc/gov/tno/nlp/Global.java

```diff
@@ -0,0 +1,43 @@
+package ca.bc.gov.tno.nlp;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Global {
```

Comment:
A way to share variables between events.  I don't love this at all, but don't know what else to do with Java.